src/Controller/CRUDController.php

```diff
@@ -306,6 +308,22 @@ public function editAction($id = null)
             throw $this->createNotFoundException(sprintf('unable to find the object with id: %s', $id));
         }
 
+        if ($parentAdmin = $this->admin->getParent()) {
```

Comment:
This might be not correct, because sometimes you could edit a child admin but without being related to the parent.

Example:

I have Pages related to a Book. I might be editing a Page of a concrete book, Or I could be editing a Page because I can access to them directly.

Reply:
Nop, it returns null if you directly go to edit the page entity